src/base/bittorrent/session.h

```diff
@@ -468,11 +468,10 @@ namespace BitTorrent
 
         // TorrentHandle interface
         void handleTorrentShareLimitChanged(TorrentHandle *const torrent);
-        void handleTorrentNameChanged(TorrentHandle *const torrent);
-        void handleTorrentSavePathChanged(TorrentHandle *const torrent);
-        void handleTorrentCategoryChanged(TorrentHandle *const torrent, const QString &oldCategory);
-        void handleTorrentTagAdded(TorrentHandle *const torrent, const QString &tag);
-        void handleTorrentTagRemoved(TorrentHandle *const torrent, const QString &tag);
+        void handleTorrentNameChanged(TorrentHandle *const torrent, const QString &torrentName);
```

Comment:
TorrentHandle::name() doesn't expose the same thing. Nothing does.

Reply:
I have commented not only on this particular method, but all similar changes.
As for this particular case, I'd better change the handling of the torrent name inside TorrentHandle class.

Reply:
Personally, I don't see this as a problem.
Do you want to provide a diff that implements this in the "correct" way?

Reply:
>Do you want to provide a diff that implements this in the "correct" way?

Then it will be easier to fix such things in the form of separate "prerequisite" PRs.